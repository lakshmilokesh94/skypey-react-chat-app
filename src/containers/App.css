.App{
  display: flex;
  height:100%;
  color:rgb(156, 156, 252);
}