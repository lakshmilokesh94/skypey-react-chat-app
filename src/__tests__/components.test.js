import React from 'react';
import renderer from 'react-test-renderer';
import App from "../containers/App";

describe('components', function() {
    describe('<App />', function() {
        it('renders correctly', function() {
            var tree = renderer.create(<App />).toJSON();
            //expect(tree).toMatchSnapshot();
          });
    });
  });

