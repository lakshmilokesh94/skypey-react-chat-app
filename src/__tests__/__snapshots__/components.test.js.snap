// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <App /> renders correctly 1`] = `
<div
  className="App"
>
  <aside
    className="Sidebar"
  >
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/markwienands/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Daisha Boyer
          </p>
        </div>
        <div
          className="User__details-status"
        >
          What we don't know for sure is whether or not a tidy kitten is a turtle of the mind.
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/adhiardana/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Dorthy Lehner
          </p>
        </div>
        <div
          className="User__details-status"
        >
          Before bananas, camels were only lemons.
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/mattbilotti/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Alvina Berge
          </p>
        </div>
        <div
          className="User__details-status"
        >
          The literature would have us believe that a plucky fig is not but a pear.
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/javorszky/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Einar Farrell
          </p>
        </div>
        <div
          className="User__details-status"
        >
          This is not to discredit the idea that placid elephants show us how cheetahs can be cats.
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/chandlervdw/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Karianne Kuphal
          </p>
        </div>
        <div
          className="User__details-status"
        >
          Some assert that before cows, kittens were only spiders.
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/ryanmclaughlin/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Gust Kerluke
          </p>
        </div>
        <div
          className="User__details-status"
        >
          They were lost without the talented fly that composed their cat.
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/jlsolerdeltoro/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Yasmeen Harber
          </p>
        </div>
        <div
          className="User__details-status"
        >
          One cannot separate elephants from broad-minded owls.
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/romanbulah/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Mrs. Kenny Barrows
          </p>
        </div>
        <div
          className="User__details-status"
        >
          Cheetahs are willing dogs.
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/lowie/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Marco Cartwright
          </p>
        </div>
        <div
          className="User__details-status"
        >
          A strawberry can hardly be considered a succinct kangaroo without also being a rat?
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
    <div
      className="User"
      onClick={[Function]}
    >
      <img
        alt="pic"
        className="User__pic"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/oscarowusu/128.jpg"
      />
      <div
        className="User__details"
      >
        <div
          className="User__details-name"
        >
          <p>
            Kip Boehm
          </p>
        </div>
        <div
          className="User__details-status"
        >
          Waking to the buzz of the alarm clock, the literature would have us believe that an eminent panda is not but a chimpanzee!
        </div>
      </div>
      <button
        className="User__remove-button"
        onClick={[Function]}
      >
        X
      </button>
    </div>
  </aside>
  <main
    className="Main"
  >
    <div
      className="Empty"
    >
      <p
        className="Empty__name"
      >
        Welcome,
        Reese
      </p>
      <img
        alt="pic"
        className="Empty__img"
        src="https://s3.amazonaws.com/uifaces/faces/twitter/hanna_smi/128.jpg"
      />
      <p
        className="Empty__status"
      >
        Status:
        A scorpion is a charming currant.
      </p>
      <button
        className="Empty__btn"
      >
        Start Conversation
      </button>
      <p
        className="Empty__info"
      >
        Search for someone to start chatting with or go to Contacts to see who is available
      </p>
    </div>
  </main>
</div>
`;
